pkgs/toggle-wofi/toggle-wofi.bash

```diff
@@ -0,0 +1,7 @@
+# Anti-pattern:
+#   - No option: It might match for this script itself. For pname. And Nix long store paths make me afraid even through they are just hash strings
+#   - --exact: It only support less than 15 chars
+#   - Injecting full path with nix: It is required in both killing and executing. And might be kept in updating.
+# Depending the arguments is bit hacky, however it is easy and okay for now. I may replace this implementation with `pkill --pidfile`
+cmd='wofi --show drun --allow-images --insensitive'
+pkill --full "$cmd" || command $cmd
```

Comment:
```diff
- pkill --full "$cmd" || command $cmd
+ pkill --full "$cmd" || eval "$cmd"
```

Required to revert https://github.com/kachick/dotfiles/pull/1124/commits/9f5374c88c70628fb6b436a4d1ed840275e9ea58 fixed in https://github.com/kachick/dotfiles/commit/1a1b0be6b1c21ee785b6a4a95c58b1048c91726e